charts/dvpe-deployment-gloo/CHANGELOG.md

```diff
@@ -7,14 +7,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [2.0.1]
```

Comment:
which version should that be? I think you have to add the new version link here [2.2.0]